cosmos/migrations/0001_initial.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 3.0.8 on 2020-07-18 15:39
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+from cosmos.models.user.constants import DEPARTMENTS, NATIONALITIES, PROGRAMS
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("auth", "0011_update_proxy_permissions"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Board",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("description", models.TextField(blank=True)),
+                ("period_from", models.DateField()),
+                ("period_to", models.DateField()),
+                ("pretix_organizer", models.CharField(max_length=20)),
+                ("group", models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to="auth.Group")),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Profile",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("department", models.CharField(choices=[(name, name) for name in DEPARTMENTS], max_length=100,),),
+                ("program", models.CharField(choices=[(name, name) for name in PROGRAMS], max_length=100,),),
+                (
+                    "nationality",
+                    models.CharField(choices=[(country, country) for country in NATIONALITIES], max_length=100,),
+                ),
+                (
+                    "gender",
+                    models.CharField(
+                        choices=[("Male", "Male"), ("Female", "Female"), ("Other", "I'd rather not say")], max_length=10
+                    ),
+                ),
+                ("tue_id", models.CharField(max_length=25, verbose_name="TU/e Number")),
+                ("card_number", models.CharField(max_length=25)),
+                ("key_access", models.CharField(default="No", max_length=3)),
```

Comment:
Why not a boolean?

Reply:
wasn't aware of such thing, will change in a commit